flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -585,7 +585,6 @@ public void testPrimaryKeyValidation() {
 					"I;UA;UB;D"));
 		// pk can be defined on cdc table, should pass
 		createTableSink(pkSchema, options1);
-		createTableSink(pkSchema, options1);
```

Comment:
It should be `createTableSource(...)`.

Reply:
I just thought it was a repeat call before your comment, thanks for your comment, I will fix it soon.